server/index.js

```diff
@@ -116,6 +116,38 @@ app.post('/api/cart', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.post('/api/orders', (req, res, next) => {
+  if (!req.session.cartId) {
+    res.status(400).json({
```

Comment:
Be sure to `return` to prevent more code from executing.

Reply:
got it!! changed it!